CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # configuration-tools
 
+## Unreleased
+
+#### Removed
+
+* The function `fmapL` is removed from `Configuration.Utils.Internal`. Instead
+  the function `first` from `Data.Bifunctor` from the `base` package can be
+  used.
+
```

Comment:
By the way, the scheme I've been following for changelogs recently (here included), is this: https://keepachangelog.com/en/1.0.0/